### DISCUSSION TITLE
Is the project interested in a localisation to Swedish for MacOS?

### DISCUSSION THREAD

**Discussion by Gandoon** [transmission/transmission#5640] [Q&A] [answered]
Since I could not find one, or any discussions of one, I thought I should ask here. I have a complete localisation to Swedish for the MacOS GUI (including the QuickLook plugin). I did it mostly for myself, because I can. But I would be more than happy to contribute to the project with this. I have _not_ localised e.g. the libtransmission or command line tools, only the GUI. Thus everything is neatly contained to a `sv.lproj` which should be easy enough to audit (something I would expect for a first time contribution).

If this is of any interest, please let me know how to submit the code.

**Comment by GaryElshaw** [comment]
All of the Transmission translations are done through Transifex, there is some info [here](https://github.com/transmission/transmission/blob/main/docs/Translating.md).

**Reply by Gandoon** [reply]
Well, I take that as a no then… There seems to be quite a lot of hoops to jump through to get through there, no information on whether the "100% finished" Swedish translation there includes a Mac GUI localisation (that has not been incorporated). And a site-specific approval process for translators. Is it even worth the effort?

I asked because I have a finished translation that I use. In source code format, i.e. a complete `sv.lproj` with the 12 `*.strings` required. It was an offer since there is no Swedish localisation for the Mac GUI.

**Comment by mikedld** [comment, marked as answer]
Swedish translation on Transifex is fully finished and reviewed. My fault it didn't make it into 4.0 but there were quite a few issues as is (with switch to autolayout) so I'm even a bit glad I didn't add new languages to the mix. Depending on the release timeline, I'll see to it that it's added to 4.1 or 4.2.

We're not accepting translation contributions as patches or files, so if using Transifex (or another platform, as we're also considering Weblate per community request) is a dealbreaker for you then it just won't work, sorry.

**Comment by Gandoon** [comment]
Not really a dealbreaker, just that signing up there (no big deal really, using GitHub) requires waiting for someone to click _yes_ for me. And I have no idea how long that will take, or the internal workflow there, as I have never used it earlier.

However, if I may be so bold as to suggest that in the future at least for things like localisations, maybe you should somehow clarify which platforms are indeed 100% finished (i.e. Qt, MacOS, &c.). That should preferably be here, rather than on Transifex or whichever platform is chosen in the future. It could be done as an issue or a discussion thread. Then people who wants a localisation that is not in the current version has a better idea of whether it is worth the effort for them to join in the effort.
